frontend/src/app/ml-models/ml-model-form/ml-model-form.component.ts

```diff
@@ -309,10 +321,12 @@ export class MlModelFormComponent implements OnInit {
       labelField.key.setValue('');
```

Comment:
Not in the scope of this review but isn't there a missing closing bracket here?

Reply:
There doesn't appear to be anything missing and I'm getting no errors either in the editor nor when it gets interpreted and bundled nor in the browser after the fact.